app/views/labware/_relatives_list.html.erb

```diff
@@ -1,27 +1,31 @@
 <%# locals: (presenter:) %>
+<%
+  found_parents = presenter.labware.parents&.filter{ |parent| parent.purpose.present? }
+  found_children = presenter.labware.children&.filter{ |child| child.purpose.present? }
+%>
 <div id="relatives-information" class="list-group list-group-flush">
 
   <div class="list-group-item">
     <strong>Parents</strong>
-    <%= content_tag :span, presenter.labware.parents.size, class: "badge badge-pill badge-secondary" if presenter.labware.parents&.any? %>
+    <%= content_tag :span, found_parents.size, class: "badge badge-pill badge-secondary" %>
```

Comment:
If `presenter.labware.parents` was `nil` then this will not be an array as has been assumed here and you'd throw a no `size` method on `nil` error here.